Resources/Audio/Ambience/Antag/attributions.yml

```diff
@@ -14,3 +14,7 @@
   license: "CC0-1.0"
   copyright: "Made by https://www.youtube.com/@a-guy173"
   source: https://github.com/A-Guy173/Music/blob/main/revolution.mp3
+- files: ["emagged_borg.ogg"]
+  license: "CC0-1.0"
+  copyright: "Made by @ps3moira on github"
+  source: https://www.youtube.com/watch?v=4-R-_DiqiLo
```

Comment:
I'm not sure if youtube videos count as good primary sources. There is no indication anywhere of the license used. I think this sound can be found somewhere either in ss13 builds, and make a more correct reference to the license and source.

Reply:
I made this as part of a request, allegedly it's self made and I was just given the video
and rolled with it.
Youtube videos have a date, user accounts, and are owned by a third party, so if it's got
the earliest time and date stamp to show it's the source it should be as legit as a reference 
to an SS13 repo?
The requester didn't request a specific license so I copy pasted creative commons.
Either way I'll follow up on all that.

Reply:
default license is not public domain its all rights reserved

Reply:
I literally just copy pasted the next line closest to my cursor at that moment,
I'm not a lawyer and didn't care and neither did the creator seemingly
but you're right, sane defaults and all that.
I'll watch out!

Reply:
I think you should publish the sound on freesound.com, specify the license there, and leave a link to this site here.

Reply:
Sure, I'll see to it.

Reply:
Update on this, 
I consulted the creator about it.
They already have it uploaded, they can verify they're the creator, 
license info will be put in the youtube description.
I don't think it's 100% professional, and youtube not being open source
could theoretically be an issue?
Is that the reason?
idek, but seems good enough to me anyways.
If a reviewer takes issue with it, then we'll upload it somewhere else instead
as suggested!